integer/src/ibig.rs

```diff
@@ -68,6 +70,23 @@ use crate::{
 pub struct IBig(pub(crate) Repr);
 
 impl IBig {
+    #[inline]
+    pub fn div_rem_floor_ref(&self, rhs: &IBig) -> (IBig, IBig) {
+        let div = self.div(rhs);
+        let rem = self.rem(rhs);
+
+        (div, rem)
+    }
+
+    #[inline]
+    pub fn is_odd(&self) -> bool {
```

Comment:
Is it necessary to have a standalone function for this? You can test it by taking the first bit, which is very fast.